servicetalk-data-protobuf/src/main/java/io/servicetalk/data/protobuf/ProtobufSerializerFactory.java

```diff
@@ -54,6 +60,18 @@ private ProtobufSerializerFactory() {
         return serializerMap.computeIfAbsent(parser, parser2 -> new ProtobufSerializer<>((Parser<T>) parser2));
     }
 
+    /**
+     * Get a {@link SerializerDeserializer}.
+     * @param clazz Used to obtain a {@link Parser} which is used to serialize and deserialize.
+     * @param <T> The type to serialize and deserialize.
+     * @return a {@link SerializerDeserializer}.
+     */
+    public <T extends MessageLite> SerializerDeserializer<T> serializerDeserializer(Class<T> clazz) {
+        @SuppressWarnings("unchecked")
+        final Parser<T> parser = (Parser<T>) parserMap.get(clazz);
+        return parser == null ? serializerDeserializer(newParser(clazz)) : serializerDeserializer(parser);
+    }
+
     /**
      * Get a {@link StreamingSerializerDeserializer} which supports &lt;VarInt length, value&gt; encoding as described
      * in <a href="https://developers.google.com/protocol-buffers/docs/techniques">Protobuf Streaming</a>.
```

Comment:
This explanation would also be helpful to accompany the top-level mime-type docs.

Reply:
Can you clarify which specific mime types you are referring to? The new types include links to the specific encoding they refer to:

```java
     * {@code "application/x-protobuf-var-int"}
     * <a href="https://developers.google.com/protocol-buffers/docs/encoding">base 128 VarInt protobuf encoding</a>.
     * Should be used with {@link Publisher} to stream data.
```